### ISSUE TITLE
No support in Sonoma 14.0

### ISSUE THREAD

**Issue by itsmeagainbruh** [chrstphrknwtn/grid-clock-screensaver#10]
Does anyone experience the issue of getitng a black screen or the screensaver but the words wont change?

**Comment by jihankim90** [comment]
same here

**Comment by maximbeekenkamp** [comment]
TLDR:
You will need to force open the app, to get around the malware protection on macOS, and then restart.

On apple silicon, thats done by control clicking open the file in finder and then also going to Privacy and Security settings and selecting Open Anyway. Assuming you've already tried to control click there should be a pop-up in that section of settings asking if you want to change the security restrictions for that specific app to Open Anyway. At this point if you try to activate the screen saver you will still just receive a black screen, but if you restart your Mac it should work fine.

https://support.apple.com/en-gb/guide/mac-help/mh40616/mac

**Comment by itsmeagainbruh** [comment]
> TLDR: You will need to force open the app, to get around the malware protection on macOS, and then restart.
> 
> On apple silicon, thats done by control clicking open the file in finder and then also going to Privacy and Security settings and selecting Open Anyway. Assuming you've already tried to control click there should be a pop-up in that section of settings asking if you want to change the security restrictions for that specific app to Open Anyway. At this point if you try to activate the screen saver you will still just receive a black screen, but if you restart your Mac it should work fine.
> 
> https://support.apple.com/en-gb/guide/mac-help/mh40616/mac

still doesnt work. it still shows as a black screen and in the privacy section it did not have the open anyway option

**Comment by restacksyj** [comment]
Try this! 
Click on the downloaded saver. It will prevent you to install it and display two options 'Move to Bin' and 'Cancel'. Click on cancel and immediately go to System settings -> Privacy and Security. Scroll down and in the security section, you will see an option Open Anyway. Then, the grid saver gets installed on your system. Now you to screensaver settings and scroll down till `Other`, there you will find the grid clock screen saver.

**Comment by supermueller** [comment]
Version 0.0.2 works fine for me on Sonoma.

**Comment by omgj35** [comment]
> > TLDR: You will need to force open the app, to get around the malware protection on macOS, and then restart.
> > On apple silicon, thats done by control clicking open the file in finder and then also going to Privacy and Security settings and selecting Open Anyway. Assuming you've already tried to control click there should be a pop-up in that section of settings asking if you want to change the security restrictions for that specific app to Open Anyway. At this point if you try to activate the screen saver you will still just receive a black screen, but if you restart your Mac it should work fine.
> > https://support.apple.com/en-gb/guide/mac-help/mh40616/mac
> 
> still doesnt work. it still shows as a black screen and in the privacy section it did not have the open anyway option

Did you ever get this to work? I'm trying to run it on my MBP M3 Pro running Mac OS Sonoma 14.4.1 

:)
